control-plane/subcommand/tls-init/command_test.go

```diff
@@ -560,4 +588,82 @@ MHcCAQEEINVuQ1fmOWH5HDG7wEqB1KObSs7q26czY7P+WLhtLDZPoAoGCCqGSM49
 AwEHoUQDQgAEu6FUB4WbJpJmXe9cXC2vf9xlvLB5aj9lfAzg4uPL+yXHUpHMYyZy
 tdsZa0jHMVkDIrNoL8nmxu6X578xNY304w==
 -----END EC PRIVATE KEY-----`
```

Comment:
nit: could we rename the original consts to be `caKeyEC` and `caCertEC` for consistency?